### PR TITLE
Create .readthedocs.yml

### PR DESCRIPTION
**Pull request by oncleben31**
For building the documentation on https://readthedocs.org/

### CODE REVIEW

**Review by Quentame — APPROVED** [review]
LGTM

File: .readthedocs.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+version: 2
+sphinx:
+  configuration: docs/conf.py
+formats: all
+python:
+  version: 3.8
+  install:
+    - requirements: docs/requirements.txt
```
Comment (Quentame) [inline .readthedocs.yml:8]: `docs/requirements.txt` does not exists

Comment (oncleben31) [reply .readthedocs.yml:8]: Nice catch. File added

**Review by Quentame — COMMENTED** [review]
Nice :+1:

**Comment by oncleben31** [comment]
Contribute to #18

**Pull request merged**